stock_lock_lot/i18n/stock_lock_lot.pot

```diff
@@ -151,3 +151,20 @@ msgstr ""
 msgid "or"
```

Comment:
Please remove this file, as Transifex exporter regenerate it automatically and it can confuse.